pkg/kapis/resources/v1alpha3/handler.go

```diff
@@ -22,22 +22,8 @@ func New(factory informers.InformerFactory) *Handler {
 	}
 }
 
-func (h Handler) handleGetNamespacedResource(request *restful.Request, response *restful.Response) {
-	resource := request.PathParameter("resources")
-	namespace := request.PathParameter("namespace")
-	name := request.PathParameter("name")
-
-	result, err := h.namespacedResourceGetter.Get(resource, namespace, name)
-	if err != nil {
-		api.HandleInternalError(response, nil, err)
-		return
-	}
-
-	response.WriteHeaderAndEntity(http.StatusOK, result)
-}
-
-// handleListNamedResource retrieves namespaced scope resources
-func (h Handler) handleListNamespacedResource(request *restful.Request, response *restful.Response) {
+// ListResource retrieves resources
+func (h Handler) ListResource(request *restful.Request, response *restful.Response) {
```

Comment:
Unnecessary exported function, and what's wrong with the function name of handleGetNamespacedResource or handleListNamespacedResource?

Reply:
agree